Ansible-security.md

```diff
@@ -0,0 +1,10 @@
+
+1. Ansible Server:
+    - Where is your ansible server located? 
+    - Who has access to it? 
+    - How do you minimize the access to the ansible server to a limited number of users.
+2. Ansible Vault:
+    - Do you use ansible-vault to encrypt the ansible playbook? 
+    - Where is the ansible vault password stored?
+3. Is SSH allowed only from the specific source address to the ansible server?
```

Comment:
Technology neutrality here would be better, I think. Something like:

What network-level security solution is in place for safeguarding access to the Ansible server (e.g., SSH, VPN)?

Reply:
Thanks  Lars !! i have done the required changes in the checklist.